### ISSUE TITLE
istio-1.0.5 - Bookinfo app - unable to access to the app via the ingress-gateway

### ISSUE THREAD

**Issue by blkwtch** [istio/istio#12057]
**Describe the bug**
curl -o /dev/null -s -w "%{http_code}\n" http://${GATEWAY_URL}/productpage ==>000
**Expected behavior**
curl -o /dev/null -s -w "%{http_code}\n" http://${GATEWAY_URL}/productpage ==>200


**Steps to reproduce the bug**
when installating the all the crds and the bookinfo app yaml file the target configuration is not enabled where all the sidecar proxies communicate with the ingress gateway to forward traffic/requests 

**Version**
{{ What version of Istio and Kubernetes are you using? Use `istioctl version` and `kubectl version` }}
kubectl  "v1.13.3"
istio-1.0.5 

**Installation**
I followed the installation process as described istio's website https://istio.io/docs/examples/bookinfo/

**Environment**
host: virtual machine using vagrant
Using Minikube with --vm-driver=none (docker)
minikube version: v0.34.1
kubectl  "v1.13.3"

**Cluster state**

vagrant@ubuntu-bionic:~/istio-1.0.5$ kubectl get pods                                                                                                                                                                                           NAME                              READY   STATUS    RESTARTS   AGE                                                                                                                                                                              details-v1-57994694dd-tgj5x       2/2     Running   0          30m                                                                                                                                                                              productpage-v1-5d4d6f9c98-hgjwz   2/2     Running   0          30m                                                                                                                                                                              ratings-v1-68c44bdc8d-kq2k9       2/2     Running   0          30m                                                                                                                                                                              reviews-v1-85cc695989-l4kbb       2/2     Running   0          30m                                                                                                                                                                              reviews-v2-74d79b74dd-fhhqw       2/2     Running   0          30m                                                                                                                                                                              reviews-v3-665d5bc57f-tg67h       2/2     Running   0          30m
--



vagrant@ubuntu-bionic:~/istio-1.0.5$ kubectl get services                                                                                                                                                                                       NAME          TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE                                                                                                                                                                         details       ClusterIP   10.109.123.204           9080/TCP   31m                                                                                                                                                                         kubernetes    ClusterIP   10.96.0.1                443/TCP    34m                                                                                                                                                                         productpage   ClusterIP   10.106.108.173           9080/TCP   31m                                                                                                                                                                         ratings       ClusterIP   10.99.7.213              9080/TCP   31m                                                                                                                                                                         reviews       ClusterIP   10.96.10.35              9080/TCP   31m
--



vagrant@ubuntu-bionic:~/istio-1.0.5$ kubectl get pods -n istio-system                                                                                                                                                                           NAME                                      READY   STATUS              RESTARTS   AGE                                                                                                                                                            grafana-59b8896965-bwthg                  1/1     Running             0          33m                                                                                                                                                            istio-citadel-856f994c58-w7d25            1/1     Running             0          33m                                                                                                                                                            istio-cleanup-secrets-htdnf               0/1     Completed           0          33m                                                                                                                                                            istio-egressgateway-54797bd4-8866g        1/1     Running             0          33m                                                                                                                                                            istio-galley-7665f65c9c-4ptdt             1/1     Running             0          33m                                                                                                                                                            istio-grafana-post-install-j29qk          0/1     Completed           0          33m                                                                                                                                                            istio-ingressgateway-7768dfb7cf-tkb7x     1/1     Running             0          33m                                                                                                                                                            istio-pilot-5945454d64-t57pj              2/2     Running             0          33m                                                                                                                                                            istio-policy-66b4c8b455-qjl6d             2/2     Running             0          33m                                                                                                                                                            istio-security-post-install-lzcq2         0/1     Completed           0          33m                                                                                                                                                            istio-sidecar-injector-768c79f7bf-zsz6z   1/1     Running             0          33m                                                                                                                                                            istio-telemetry-64bbbfbfd7-h6hgl          2/2     Running             0          33m                                                                                                                                                            istio-tracing-6b994895fd-tfcg9            1/1     Running             0          33m                                                                                                                                                            prometheus-76b7745b64-6qbs6               1/1     Running             0          33m                                                                                                                                                            servicegraph-5c4485945b-wxmzt             0/1     ImageInspectError   0          33m
--



NAME                     TYPE           CLUSTER-IP       EXTERNAL-IP   PORT(S)                                                                                                     AGE                                                          grafana                  ClusterIP      10.108.62.53             3000/TCP                                                                                                    33m                                                          istio-citadel            ClusterIP      10.98.162.202            8060/TCP,9093/TCP                                                                                           33m                                                          istio-egressgateway      ClusterIP      10.105.187.89            80/TCP,443/TCP                                                                                              33m                                                          istio-galley             ClusterIP      10.108.118.101           443/TCP,9093/TCP                                                                                            33m                                                          istio-ingressgateway     LoadBalancer   10.107.30.227    10.0.2.15     80:31380/TCP,443:31390/TCP,31400:31400/TCP,15011:32565/TCP,8060:32352/TCP,15030:31930/TCP,15031:31748/TCP   33m                                                          istio-pilot              ClusterIP      10.100.211.187           15010/TCP,15011/TCP,8080/TCP,9093/TCP                                                                       33m                                                          istio-policy             ClusterIP      10.103.36.135            9091/TCP,15004/TCP,9093/TCP                                                                                 33m                                                          istio-sidecar-injector   ClusterIP      10.103.122.26            443/TCP                                                                                                     33m                                                          istio-telemetry          ClusterIP      10.106.8.243             9091/TCP,15004/TCP,9093/TCP,42422/TCP                                                                       33m                                                          jaeger-agent             ClusterIP      None                     5775/UDP,6831/UDP,6832/UDP                                                                                  33m                                                          jaeger-collector         ClusterIP      10.111.209.2             14267/TCP,14268/TCP                                                                                         33m                                                          jaeger-query             ClusterIP      10.108.114.228           16686/TCP                                                                                                   33m                                                          katacoda-grafana         ClusterIP      10.110.53.122    10.0.2.15     3000/TCP                                                                                                    32m                                                          katacoda-jaeger-query    ClusterIP      10.103.152.51    10.0.2.15     16686/TCP                                                                                                   32m                                                          katacoda-prometheus      ClusterIP      10.108.57.49     10.0.2.15     9090/TCP                                                                                                    32m                                                          katacoda-servicegraph    ClusterIP      10.109.98.90     10.0.2.15     8088/TCP                                                                                                    32m                                                          prometheus               ClusterIP      10.101.213.160           9090/TCP                                                                                                    33m                                                          servicegraph             ClusterIP      10.103.230.145           8088/TCP                                                                                                    33m                                                          tracing                  ClusterIP      10.105.48.59             80/TCP                                                                                                      33m                                                          zipkin                   ClusterIP      10.97.134.56             9411/TCP                                                                                                    33m
--



vagrant@ubuntu-bionic:~/istio-1.0.5$ istioctl proxy-status -n istio-system                                                                                                                                                                      Stderr when execute [/usr/local/bin/pilot-discovery request GET /debug/syncz ]: gc 1 @0.016s 19%: 1.1+1.0+2.7 ms clock, 3.3+0.11/1.1/0.39+8.1 ms cpu, 4->4->1 MB, 5 MB goal, 3 P                                                                gc 2 @0.033s 17%: 0.006+1.5+1.6 ms clock, 0.018+0.39/0.63/2.1+4.9 ms cpu, 4->4->2 MB, 5 MB goal, 3 P                                                                                                                                                                                                                                                                                                                                                                                            PROXY     CDS     LDS     EDS     RDS     PILOT     VERSION
--



vagrant@ubuntu-bionic:~/istio-1.0.5$ istioctl proxy-config clusters productpage-v1-5d4d6f9c98-hgjwz                                                                                                                                             SERVICE FQDN         PORT     SUBSET     DIRECTION     TYPE                                                                                                                                                                                     prometheus_stats     -        -          -             STATIC                                                                                                                                                                                   xds-grpc             -        -          -             STRICT_DNS                                                                                                                                                                               zipkin               -        -          -             STRICT_DNS
--

**Comment by fredlb** [comment]
I had this issue as well, very similar setup.
I was initially trying with `--kubernetes-version=1.10.0`, which would stall during the minikube setup phase.
Then I tried 1.13.4, which worked setting up but produced the error above.
Now I dropped down to kubernetes version 1.11.0, and it works.

I am a complete kubernetes and istio beginner, so I am not sure what is at fault here, but if you want me to provide further info, just reach out with instructions.
